.prospector/opencraft.yml

```diff
@@ -5,7 +5,7 @@ dodgy:
   enable: []
   options: {}
   run: null
-ignore-paths: ['migrations', 'deploy/', 'venv/']
+ignore-paths: ['migrations', 'deploy/', 'venv/', 'frontend/']
```

Comment:
I added this because there is no Python code that we have written inside the `frontend` folder,